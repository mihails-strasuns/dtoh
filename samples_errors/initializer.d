module initializer;

struct S
{
    int x = 42;
}

extern(C) __gshared S try_me;
