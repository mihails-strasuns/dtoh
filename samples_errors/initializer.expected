samples_errors/initializer.d(5) [initializer.S]
	C does not support explicit field initializers in structs
